website/docs/d/nat_gateway.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "aws"
+page_title: "AWS: aws_nat_gateway"
+sidebar_current: "docs-aws-datasource-nat_gateway"
+description: |-
+    Provides details about a specific Nat Gateway
+---
+
+# aws\_nat\_gateway
```

Comment:
Antislashes are not needed anymore in titles, as per one of @sethvargo 's contributions :)